nginx/default.toml

```diff
@@ -34,5 +40,18 @@ tcp_nodelay = "on"
 # http.keepalive_timeout: Timeout on client connection keepalive, in seconds. Default = 75
 keepalive_timeout = 60
 
+# http.server_names_hash_bucket_size: Sets the bucket size for the server names hash tables.
+# The default value depends on the size of the processor’s cache line. Default = 32
+server_names_hash_bucket_size = 64
```

Comment:
You are probably right, but then there are dozens and dozens of Nginx directives that ought to be listed here if the intention is to be entirely rigorous.  In my experience 32 is absurdly restrictive, it causes a whole bunch of nuisance when you hit that ceiling, while at the same you can increase that number dramatically with zero impact on resources or performance.